gcp_variant_transforms/pipeline_common.py

```diff
@@ -154,6 +156,21 @@ def _get_file_names(input_file):
     return contents
 
 
+def create_file_path_to_file_hash_map(input_patterns, project):
```

Comment:
As discussed offline, this doesn't support local files. I agree with your assessment that at least a graceful failure is required in case that a local input pattern had been supplied.

@kemp-google we should decide whether we want to provide support for the local files.

Reply:
If we hash the whole file name using our `hashing_util.generate_int64_hash_code` and do not rely on `file_blob.md5_hash` then don't need to limit this to GCS files only, right?
I think I prefer that because:
 * we are able to control the range of the hashed values (in case we need to perform partitioning)
 * we will be using farm_fingerprint hashing consistently every where.